.github/workflows/my-ci-pipline.yml

```diff
@@ -18,6 +18,39 @@ jobs:
     - run: docker build --target test --tag todo-app:test . 
     - run: docker run todo-app:test   
 
+  production:
+    name: Production
+    needs: build
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master' && github.event_name == 'push' 
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Setting up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Authenticating DockerHub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Pushing to DockerHub
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        target: production
+        push: true
+        tags: ${{ secrets.DOCKERHUB_USERNAME }}/todoapp:latest
+        cache-from: type=registry,ref=${{ secrets.DOCKERHUB_USERNAME }}/todoapp:buildcache
+        cache-to: type=registry,ref=${{ secrets.DOCKERHUB_USERNAME }}/todoapp:buildcache,mode=max
+
+    - name: Deploying to Azure
+      run: curl -dH -X POST ${{secrets.WEBHOOK_URL}}
+      
+       
+
 #I have not dont the module 8 yet. I need to use the curl command in future 
  #final step 
 #curl -dH -X POST ${{secrets.WEBHOOK_URL}}
```

Comment:
We can get rid of these now! But we should add a link to your deployed application now so that users can visit that from your Readme
(I'm assuming it's <https://jo-todo-app.azurewebsites.net/> since <https://joetodoapp.azurewebsites.net/> is currently broken, so is probably just being explored for M12!)